java/tutorial/pom.xml

```diff
@@ -41,33 +41,14 @@
             <RAY_CONFIG>${basedir}/../ray.config.ini</RAY_CONFIG>
           </environmentVariables>
           <argLine>-ea
-            -Djava.library.path=${basedir}/../../build/src/plasma:${basedir}/../../build/src/local_scheduler
+            -Djava.library.path=${basedir}/../../build/src/plasma
```

Comment:
why not package plasma lib as well?

Reply:
I made a mistake here, I thought local_scheduler native lib is not required any more.

but, raylet java client is placed in the local_scheduler native lib! it should be fixed in the future.

I should add the local_scheduler directory back here...